content/trusted-content/official-images/contributing.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contributing to Docker Official Images
+description: |
+  This article describes how Docker Official Images are created,
+  and how you can contribute or leave feedback.
+keywords: docker official images, doi, contributing, upstream, open source
+---
+
+Docker, Inc. sponsors a dedicated team that's responsible for reviewing and
+publishing all content in Docker Official Images. This team works in
+collaboration with upstream software maintainers, security experts, and the
+broader Docker community.
+
+While it's preferable to have upstream software authors maintaining their
+Docker Official Images, this isn't a strict requirement. Creating
+and maintaining images for Docker Official Images is a collaborative process.
+It takes place [openly on GitHub](https://github.com/docker-library/official-images)
+where participation is encouraged. Anyone can provide feedback, contribute
+code, suggest process changes, or even propose a new Official Image.
+
+> **Note**
+>
+> Docker Official Images are the intellectual property of Docker.
```

Comment:
So it doesn't get lost, we do have a community request to dig more into this: https://github.com/docker-library/official-images/issues/14346